browser-extension/src/addTournesolRecommendations.js

```diff
@@ -123,6 +123,25 @@ const getLocalizedActionError = () => {
   );
 };
 
+const displayElement = (element) => {
+  const classes = element.className.split(' ');
+
+  if (!classes.includes('displayed')) {
+    classes.push('displayed');
+    element.className = classes.join(' ');
+  }
```

Comment:
[`classList` ](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList )is what we need: 
Can be simplified into
```suggestion
  element.classList.add('displayed');
```